tests/uniqueTest.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+/**
+ * @covers Dash\unique
+ */
+class uniqueTest extends PHPUnit_Framework_TestCase
+{
+	/**
+	 * @dataProvider cases
+	 */
+	public function test($iterable, $expected)
+	{
+		$this->assertSame($expected, Dash\unique($iterable));
```

Comment:
We also need to test the `distinct()` alias here too. See the tests for `first()` for guidance.

Reply:
Good call.  Done.